workers/data_refinery_workers/processors/smasher.py

```diff
@@ -495,6 +497,10 @@ def _smash(job_context: Dict) -> Dict:
                         ks_test = rlang("ks.test")
 
                         set_seed(123)
+                        n = ncol(reso)
+                        m = 2
+                        if n < m:
```

Comment:
This could just be `if ncol(reso) < 2:` right?

Reply:
Had to add a [0] to this anyway